src/MainPage.js

```diff
@@ -46,36 +46,35 @@ function MainPage({
 const MainPageStyled = styled.div`
     display: grid;
 
-    h1 {
-        margin-bottom: 25px;
-        font-size: 32px;
-        line-height: 1.4;
-    }
-
-    h2 {
-        font-size: 20px;
-        line-height: 1.3;
-        font-weight: 500;
-        color: var(--gray);
-        width: 75%;
-        margin-bottom: 42px;
+    .logo {
```

Comment:
Do you still need ``.logo``?
You could set the width in line 73

Reply:
Like that I cant place it in the center, but did this solution instead:
```suggestion
    const VirusRedStyled = styled(VirusRed)`
    width: 120px;
`
```